packages/perseus/src/widgets/interactive-graphs/graphs/components/movable-line.tsx

```diff
@@ -240,6 +252,54 @@ const Line = (props: LineProps) => {
     );
 };
 
+export const getMovableLineKeyboardConstraint = (
```

Comment:
I love that this fixes the ray overlapping points issue!

(It still happens with mouse, but that is irrelevant here and arguably not an issue.)